### DISCUSSION TITLE
Cant delete templates.

### DISCUSSION THREAD

**Discussion by StasABerg** [semaphoreui/semaphore#1326] [Q&A] [answered]
Im trying to delete a template but get prompted with and error that the I cant delete it due to ''resources below being in use"
the resource is the template Im trying to delte
![image](https://github.com/ansible-semaphore/semaphore/assets/114340468/e3ae0eae-34fe-42aa-b984-0a12d8d1ed26)

Using version 2.8.89

**Comment by theapu** [comment, marked as answer]
I got the same problem. I had set cron for the template. Afte removing cron job line from the cron field I was able to delete the template.

**Reply by fiftin** [reply]
Thank you @theapu

**Comment by StasABerg** [comment]
Yeah you need to empty the cronjob field in order to delete the task.
A bit weird solutions but works!
Thanks @theapu

**Comment by tumbl3w33d** [comment]
What is the rationale for this being a discussion and not a bug ticket? Can't think of any other software where you would need to manually resolve constraints/ensure integrity by following a certain order of deleting things. It at least should tell you that you need to delete that cron to be able to delete the template, but that would still be weird.
